conversation-api/api-reference/topics.md

```diff
@@ -125,10 +131,10 @@ exit()
 
 ### Query Params
 Parameter | Required | Value |Description|
---------- | --------- | ------- | -------
+--------- | --------- | ------- | ------- | -------
 ```sentiment```| No | true | Give you sentiment analysis on each topic in conversation.
 ```parentRefs```| No | true | Gives you [topic hierarchy](/docs/concepts/topic-hierarchy).
```

Comment:
@amritesh-singh 


Replace the following:

"Value" to "Type"
"true" to "Boolean"
Add another column and call it "Value" and add true or false`.